src/redux/configureStore.js

```diff
@@ -0,0 +1,17 @@
+import { configureStore, combineReducers } from '@reduxjs/toolkit';
+import { applyMiddleware, compose } from 'redux';
+import thunk from 'redux-thunk';
+import rocketReducer from './rockets/rockets';
+import missionReducer from './missions/MissionSlice';
+
+const rootReducers = combineReducers({
+  missions: missionReducer,
+  rockets: rocketReducer,
+});
+
+const store = configureStore({
+  reducer: rootReducers,
+});
+export default store;
+const composeEnhancers = compose;
+composeEnhancers(applyMiddleware(thunk));
```

Comment:
- [ ] After installing redux-logger, please add it to your middleware and test if it is working as required 🏄